exporter/tencentcloudlogserviceexporter/README.md

```diff
@@ -15,19 +15,33 @@ This exporter supports sending OpenTelemetry log data to [LogService](https://cl
 
 ```yaml
 receivers:
-  examplereceiver:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: ":4317"
 
 exporters:
   tencentcloud_logservice:
-    endpoint: "ap-beijing.cls.tencentcs.com"
-    logset: "test-logset"
-    topic: "test-topic"
-    secret_id: "secret-id"
-    secret_key: "secret-key"
+      # LogService's Region, https://cloud.tencent.com/document/product/614/18940
+      # set cls.{region}.tencentcloudapi.com, eg cls.ap-beijing.tencentcloudapi.com;
+    region: "ap-beijing"
+    # LogService's LogSet ID
+    logset: "demo-logset"
+    # LogService's Topic ID
+    topic: "demo-topic"
+    # TencentCloud secret id
+    secret_id: "demo-secret-id"
+    # TencentCloud secret key
+    secret_key: "demo-secret-key"
 
 service:
   pipelines:
-    traces:
-      receivers: [examplereceiver]
+    logs:
+      receivers: [otlp]
       exporters: [tencentcloud_logservice]
-```
\ No newline at end of file
+```
+
+# Changelog
```

Comment:
I believe @codeboten meant the main changelog from this repository:

https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/87ba2fa130a3b1353d717499e3b62c7bea4924a0/CHANGELOG.md?plain=1#L3-L7

Reply:
Get.

Reply:
I would remove this changelog from the local readme, but it's your call :-) I'll merge it as-is once the CI allows, feel free to send another PR to remove it if you want.